kart/diff_util.py

```diff
@@ -30,9 +31,13 @@ def __init__(self, repo, all_ds_paths=None):
         self.all_ds_paths = all_ds_paths
 
     @property
-    def table_working_copy(self):
+    def working_copy(self):
         assert not self.is_bare
-        return self.repo.working_copy.tabular
+        return self.repo.working_copy
+
+    @property
+    def wc(self):
```

Comment:
IMO we don't need these aliases. Just use `working_copy`. Adding extra aliases isn't actually helping with readability